src/docs/reference/modules/Recipes/README.md

```diff
@@ -325,12 +325,27 @@ The Deployment step allows you to create a deployment plan with deployment steps
     }
 ```
 
+### CustomSettings Step
+
+The CustomSettings step allows you to populate your custom settings with initial values.
+
+```json
+    {
+      "name": "custom-settings",
+      "BlogSettings": {
+        "ContentType": "BlogSettings",
```

Comment:
It's actually expected this will be the full json of a content item, not the limited parts you have included here.
This would work, but for an example, but for an example it would be better if you could can create one using the `CustomSettings` deployment step.

(this is missing any fields for example)

Reply:
@deanmarcussen sorry but I don't undestand last sentence. So I should also add also `ContentItemId` and some custom fields (i.e. TextField) right?

Reply:
No prob. 
Go to import/export -> deployment plans.
Make a deployment plan with the `CustomSetting` in it.
Execute the plan.
You will see the full format of the custom setting.

It would not hurt if somewhere on the docs we explained that you can by and large see and understand a recipe steps format, by making a deployment plan. (so much easier than trying to guess at it!)